chrome/manifest.dev.json

```diff
@@ -41,5 +41,5 @@
     "*://m.youtube.com/*", 
     "*://www.youtube.com/*"
   ],
-  "content_security_policy": "default-src 'self'; script-src 'self' http://localhost:3000 https://localhost:3000 'unsafe-eval'; connect-src http://localhost:3000 https://localhost:3000 https://localhost:4003; style-src * 'unsafe-inline' 'self' blob:; img-src 'self' data:;"
+  "content_security_policy": "default-src 'self'; script-src 'self' http://localhost:3000 https://localhost:3000 'unsafe-eval'; connect-src http://localhost:3000 https://localhost:3000 https://localhost:4003; style-src * 'unsafe-inline' 'self' blob:; img-src 'self' https://img.youtube.com data:;"
```

Comment:
- `https://img.youtube.com` permission in manifest.dev.json : OK
- `https://img.youtube.com` permission in manifest.prod.json : FAIL (please add the change on manifest.prod.json)
- `https://img.youtube.com` permission in manifest.staging.json : FAIL (please add the change on manifest.staging.json)

Reply:
Well done, thanks!